Code/Tools/FBuild/FBuildCore/FBuild.cpp

```diff
@@ -742,28 +742,32 @@ bool FBuild::DisplayDependencyDB( const Array< AString > & targets ) const
 //------------------------------------------------------------------------------
 /*static*/ bool FBuild::GetTempDir( AString & outTempDir )
 {
-    #if defined( __WINDOWS__ )
+    #if defined( __WINDOWS__ ) || defined( __LINUX__ )
```

Comment:
This is a good fix. It looks like we can actually apply the same logic on OSX as well.

Reply:
Applied for __APPLE__ too. I think __APPLE__ is more reliable define than __OSX__.